drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -561,6 +566,10 @@ static void _isr(netdev_t *netdev)
                 at86rf2xx_set_state(dev, dev->idle_state);
                 DEBUG("[at86rf2xx] return to state 0x%x\n", dev->idle_state);
             }
+#ifdef MODULE_AT86RF233
```

Comment:
A quick scan of the manual tells me that the RF232 provides this register as well. There is no mention of ARET_FRAME_RETRIES in the data sheets for RF212B or RF231 though.

Reply:
I thought I checked them all, but thanks for checking. I must have missed the RF232.